src/quo2/components/tags/context_tag/view.cljs

```diff
@@ -39,16 +40,21 @@
     (trim-public-key public-key)]])
 
 (defn context-tag
-  [{:keys [text-style blur?] :as params} photo name channel-name]
-  (let [text-params {:weight :medium
-                     :size   :paragraph-2
-                     :style  (assoc text-style :justify-content :center)}]
+  [{:keys [text-style blur? no-avatar-placeholder?] :as params} photo name channel-name]
```

Comment:
Nothing too important, but the flag `no-avatar-placeholder?` could be used in the quo preview namespace.

Reply:
Added 😄